WordPress/Classes/ViewRelated/Comments/FullScreenCommentReplyViewController.swift

```diff
@@ -14,43 +14,33 @@ fileprivate enum SuggestionsPosition: Int {
 }
 
 public class FullScreenCommentReplyViewController: EditCommentViewController, SuggestionsTableViewDelegate {
-    private struct Parameters {
-        /// Determines the size of the replyButton
-        static let replyButtonIconSize = CGSize(width: 21, height: 18)
-
-        // Static margin between the suggestions view and the text cursor position
-        static let suggestionViewMargin: CGFloat = 5
-    }
 
     /// The completion block that is called when the view is exiting fullscreen
     /// - Parameter: Bool, whether or not the calling view should trigger a save
     /// - Parameter: String, the updated comment content
     public var onExitFullscreen: ((Bool, String) -> ())?
 
     /// The save/reply button that is displayed in the rightBarButtonItem position
-    private(set) var replyButton: UIButton!
+    private(set) var replyButton: UIBarButtonItem!
 
     /// Reply Suggestions
     ///
     private var siteID: NSNumber?
     private var suggestionsTableView: SuggestionsTableView?
 
+    // Static margin between the suggestions view and the text cursor position
+    private let suggestionViewMargin: CGFloat = 5
+
     // MARK: - View Methods
     public override func viewDidLoad() {
         super.viewDidLoad()
-
-        title = NSLocalizedString("Comment", comment: "User facing, navigation bar title")
-
-        setupReplyButton()
         setupNavigationItems()
-        configureAppearance()
+        configureNavigationAppearance()
     }
 
     public override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-
-        enableRefreshButtonIfNeeded(animated: false)
-
+        refreshReplyButton()
```

Comment:
I didn't realize this was in `viewDidAppear`. I moved it to `viewWillAppear` in #17448.